Sources/ViewInspector/Inspector.swift

```diff
@@ -1,18 +1,125 @@
+    /// - Returns: The type name with its generic parameters replaced. (e.g. ``SomeTypeName<EmptyView>``)
+    static func replaceGenericParameters(inTypeName typeName: String,
+                                         withReplacement replacement: String) -> String {
+        // Check memoized value
+        if let typeNameDict = Inspector.replacedGenericParametersCache[typeName],
+           let cachedResult = typeNameDict[replacement] {
+            return cachedResult
+        }
+
+        // Compute value
+        let result = computeReplacementOfGenericParameters(inTypeName: typeName,
+                                                           withReplacement: replacement)
+
+        // Store memoized value
+        var typeNameDict = Inspector.replacedGenericParametersCache[typeName] ?? [:]
+        typeNameDict[replacement] = result
+        Inspector.replacedGenericParametersCache[typeName] = typeNameDict
+
+        return result
+    }
+
+    private static func computeReplacementOfGenericParameters(inTypeName typeName: String,
+                                                              withReplacement replacement: String) -> String {
+        guard let start = typeName.firstIndex(of: "<") else {
+            return typeName
+        }
+
+        var balance = 1
+        var current = typeName.index(after: start)
+        while balance > 0 && current < typeName.endIndex {
+            let char = typeName[current]
+            if char == "<" { balance += 1 }
+            if char == ">" {
+                guard let indexOfPreviousChar = typeName.index(current,
+                                                               offsetBy: -1,
+                                                               limitedBy: typeName.startIndex) else {
+                    return typeName
+                }
+                let previousChar = typeName[indexOfPreviousChar]
+                if previousChar == "-" {
+                    // We've found the "->" arrow for a closure type. Ignore this ">".
+                } else {
+                    balance -= 1
+                }
+            }
+            current = typeName.index(after: current)
+        }
+
+        if balance == 0 {
+            return String(typeName[..<start]) +
+            replacement +
+            replaceGenericParameters(inTypeName: String(typeName[current...]),
+                                     withReplacement: replacement)
+        }
+
+        return typeName
+    }
+
+    private static var replacedGenericParametersCache: [String : [String : String]] = [:]
+    private static var sanitizedNamespacesCache: [String: String] = [:]
+
+    private static let sanitizeNamespacePatterns = [
+        "\\.\\(unknown context at ..........\\)",
+        // This pattern may be helpful to solve issue #268.
+        // It will remain disabled until it is confirmed.
+        // https://github.com/nalexn/ViewInspector/issues/268
+        //"\\(extension in [a-zA-Z0-9]*\\)\\:",
```

Comment:
@nalexn I left this disabled since #268 is not yet resolved with a conclusion that this additional pattern helps.
I also left a comment for context.